### DISCUSSION TITLE
Breaking the SD card out to a case

### DISCUSSION THREAD

**Discussion by adam-ainsworth** [UzixLS/zx-sizif-512#27] [Q&A] [answered]
Hi All,

I have put my Sizif into a Spectrum+ case. It’s all working nicely, except the SD card isn’t accessible with the case closed. 

I bought an extender to break it out the edge of the case (after a little alteration to make it fit) but it seems that the card needs to be physically ejected to disable the DivMMC functionality. 

Does anyone have any ideas (short of not opening the case constantly) about how I can relocate the slot?

Thanks very much in advance. 

Adam

![7E7786FE-851E-4940-BD89-1A3954D423FD](https://user-images.githubusercontent.com/554151/214398860-79654bce-0f2c-4485-bf1f-9215aa665e36.jpeg)

**Comment by UzixLS** [comment, marked as answer]
It's possible to disable divmmc via magic(NMI) menu.

**Reply by adam-ainsworth** [reply]
Really? I haven’t really checked it out since I upgraded to the most recent version of the ROM. Thanks very much :-)
